src/Module.php

```diff
@@ -35,7 +35,7 @@ public function init($event)
         $serviceListener->addServiceManager(
             'FilterManager',
             'filters',
-            'Zend\ModuleManager\Feature\FilterProviderInterface',
+            FilterProviderInterface::class,
```

Comment:
@Maks3w This is being done to remove the various provider classes from zend-modulemanager; this PR (and others that @golovanov has provided today) are simply porting those providers to the relevant components, and modifying the `ServiceListener` configuration to reference the new provider.

The primary problem with the approach, however, is that if users were implementing the old interface, they now need to switch to the new one. That said, the `ServiceListener` looks for either the correct interface implementation, *or the method it defines*, so functionality *should* continue to work. The issue will be if we then remove the related interface from zend-modulemanager, as *that* situation would break existing code.